docs/source/templates/generative-supervised-llm.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Supervised LLM Fine-tuning
+type: templates
+category: Generative AI
+cat: generative-ai
+order: 903
+is_new: t
+meta_title: Create dataset for supervised LLM fine-tuning
+meta_description: Template for creating dataset for supervised LLM fine-tuning with Label Studio for your machine learning and data science projects.
+---
+
+The goal of Supervised LLM Fine-tuning is to optimize your LLM to generate responses given user-defined prompts. There
```

Comment:
Currently, line breaks in markdown are rendered as line breaks in the generated text. If that is the desired outcome of our rendering engine (which is [out of spec with the markdown standard](https://daringfireball.net/projects/markdown/syntax#precode)), then all paragraphs in the document should be on one line.